DEVELOPMENT.md

```diff
@@ -94,14 +94,26 @@ $ git checkout master
 $ git merge origin release-candidate      # merge in most recent release candidate
 $ npm install && npm run build            # Make sure to build Ceramic and install current dependencies before releasing
 $ npm adduser                             # login to npm accunt
-$ npm run publish:latest                  # create and publish the release
+$ npm run publish:release                  # create and publish the release
 $ git checkout release-candidate
 $ git merge origin master                 # Merge the new release commit back into the release-candidate branch
 $ git push origin release-candidate
 $ git checkout develop
 $ git merge origin master                 # Merge the new release commit back into the develop branch
 $ git push origin develop
 ```
-The main step that creates the release is `npm run publish:latest`. This creates the release on NPM, as well as making a git commit bumping the version and a git tag and release on github. Make sure to set the [GH_TOKEN](https://github.com/lerna/lerna/tree/master/commands/version#--create-release-type) environment variable and log into npm before you run this command.
+The main step that creates the release is `npm run publish:release`. This creates the release on NPM, as well as making a git commit bumping the version, new git tags, and a release on github. Make sure to set the [GH_TOKEN](https://github.com/lerna/lerna/tree/master/commands/version#--create-release-type) environment variable and log into npm before you run this command.
+
+After the release, don't forget to make a post in the #releases channel of the Ceramic discord to notify the community about the new release!
+
+### Hotfix release
+```
+export GH_TOKEN=<your github token>       # You need a valid github api token set to create the release on github
+$ git checkout <hotfix branch>
+$ npm install && npm run build            # Make sure to build Ceramic and install current dependencies before releasing
+$ npm adduser                             # login to npm accunt
+$ npm run publish:hotfix                  # create and publish the release
+```
+The main step that creates the release is `npm run publish:hotfix`. This creates the release on NPM, as well as making a git commit bumping the version and new git tags on github. Make sure to set the [GH_TOKEN](https://github.com/lerna/lerna/tree/master/commands/version#--create-release-type) environment variable and log into npm before you run this command.
```

Comment:
Can we add some description of what a hotfix is, and explain that whatever branch it's run against will be released as a real full release, so it should generally be done from a recent fork of `main`?